workshops/python and data visualizations/README.md

```diff
@@ -0,0 +1,325 @@
+![Stonks](https://wompampsupport.azureedge.net/fetchimage?siteId=7575&v=2&jpgQuality=100&width=700&url=https%3A%2F%2Fi.kym-cdn.com%2Fentries%2Ficons%2Ffacebook%2F000%2F029%2F959%2FScreen_Shot_2019-06-05_at_1.26.32_PM.jpg)
+
+You after you make your first data visualization^
+
+*Follow along with this workshop to gain some data science skills! You will use python to create basic graphs and visualizations with real data. You can hand draw a graph anytime you want, but why not learn how to code one! You will learn about matplotlib, pandas, and numpy, so you can create any graph or visualization you dream of!*
```

Comment:
The start of your workshop should lay out a specific thing you’re going to make, with a link &/or screenshot of the live demo, as well as the source code. Even if folks deviate from the workshop (which we hope they will) to take it in their own direction, they should be able to follow it with no changes & get exactly what you did.